### ISSUE TITLE
508 findings for Facility Locator (Mapbox upgrade) pre-launch 508 review

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#3978]
## Team Name

Facility Locator

## Product Description

Facility Locator Mapbox functionality review following version upgrade

## Staging URL and Credentials

Facility Locator is part of unauthenticated experience and does not require test users. 

## Test Plan

[#3586]

## Known Issues

[508 Issue Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=508-issue.md&title=)

**Comment by mmiddaugh** [comment]
@jenstrickland noted this will also require a performance audit

**Comment by jenstrickland** [comment]
@mmiddaugh I will do a rudimentary performance audit as part of a spot check, and if it looks problematic we can request a formal one. 🤞 it will be lickity split!

**Comment by jenstrickland** [comment]
👍 
![20200221-mapbox-upgrade-lighthouse](https://user-images.githubusercontent.com/57469/75060988-cd614200-54ad-11ea-8ca6-103caad259cf.png)

**Comment by jenstrickland** [comment]
For reference, I opened this issue https://github.com/department-of-veterans-affairs/va.gov-team/issues/6097, and assigned it to the VSP sitewide team, during the audit, as an axe-core issue was found in the page template, that heading levels should only increase by one. This is due to there only being an `<h1>` on this page, and the footer content starts with an `<h4>`. This is outside the scope of the VSA Facilities team's efforts.

**Comment by jenstrickland** [comment]
For all the screen readers, the content area of the map was silent. Only the Zoom In and Zoom Out buttons and the attribution text was read. The attribution content lacked context, and was confusing without it. 

Supplemental info, likely for the MVP UX/UI audit
- The search results associated with the search read as a wall of words, rather than framing as a list of results.
- When the Search button is selected, NVDA reads "Searching for Virginia Indiana" which I cannot figure out why.

**Comment by jenstrickland** [comment]
@mmiddaugh — this pre-launch 508 review is complete. Since this is in production, I'll send to the VA 508 office next.

**Comment by mmiddaugh** [comment]
@jenstrickland can we close this issue or do we wait for feedback from the VA 508 office?

**Comment by jenstrickland** [comment]
@mmiddaugh We typically keep these open until the issues are closed. The 508 office will be provided this epic to show the list of known issues.
